client/controllers/sourceModal.coffee

```diff
@@ -192,7 +191,11 @@ Template.sourceModal.events
     input.value = @url
     titleInput = instance.find('#title')
     titleInput.value = @subject
+    $(input).trigger('input').trigger('input')
```

Comment:
I was wondering the same. I think it was originally introduced here: https://github.com/ecohealthalliance/eidr-connect/commit/8756b6f5a23b419ab4303c58bd4fb371ae414393#diff-96f74db91476ffbdb7217d25f79ce73bR180